### PR TITLE
 🔥 remove a bunch of project and app templates

### PR DESCRIPTION
**Pull request by wei-lee**
Only the following templates are kept:

* Hello world
** native swift for ios
** native gradle for android
** cordova
** cloud app
* Form Template
** form client app
** cloud app
* Empty template
* Mbaas service template
** cloud app

@davidffrench mind taking a look?

### CODE REVIEW

File: global.json [hunk 1]
```diff
@@ -432,20 +111,6 @@
...
-        "description": "Clustered Hello World Node.js Express App which echos a username",
-        "docs": "https://raw.githubusercontent.com/feedhenry-templates/helloworld-cloud-cluster/master/README.md",
-        "repoUrl": "git://github.com/feedhenry-templates/helloworld-cloud-cluster.git",
-        "githubUrl": "https://github.com/feedhenry-templates/helloworld-cloud-cluster.git",
-        "repoBranch": "refs/heads/master",
-        "icon": "icon-cloud",
-        "category": "Sample Apps",
-        "forcedSelection": true
-      },
       {
```
Comment (davidffrench) [inline global.json]: @wei-lee This blank Android template can be removed as well

**Comment by wei-lee** [comment]
@davidffrench thanks, it's fixed now.

**Comment by wei-lee** [comment]
@davidffrench I have deployed the changes to here: https://rhmap.csteam2.skunkhenry.com/#projects/new. If you can take a quick look and if you are happy with it I will merge the PR.

**Comment by davidffrench** [comment]
All looks good to me @wei-lee . I am curious if there will be any issues when trying to clone an existing app that was created from a now deprecated template. Unfortunately, there are none on csteam2 and there seems to be an issue with cloning apps.

**Comment by wei-lee** [comment]
@davidffrench I think that should continue to work as cloning will use the content of the existing app, it should have nothing to do with the template at all. But you are right it's better to confirm.

**Pull request merged**